group_study/communication.md

```diff
@@ -19,31 +27,33 @@
 
 | Day | How | The topic of discussion |
 | --- | :-: | ----------------------- |
-|     |     |                         |
+|  16.01.2024   |  https://meet.google.com/wfz-xkgb-wzs   |  Norms                       |
 
 ## Communication Channels
```

Comment:
I think under this section if we can add slack and WhatsApp as the channels of communication and providing some clarification of how and when we should use from a particular channel.

Reply:
I will add a link to slack and that would be enough I guess. Describing every channel in slack... that would be a bit time consuming.